src/share/instruments/Casio_CDP-S360.idf

```diff
@@ -0,0 +1,767 @@
+    <Patch name="665 GM BIRD" lbank="0" hbank="0" prog="123" />
+    <Patch name="666 GM TELEPHONE" lbank="0" hbank="0" prog="124" />
+    <Patch name="667 GM HELICOPTER" lbank="0" hbank="0" prog="125" />
+    <Patch name="668 GM APPLAUSE" lbank="0" hbank="0" prog="126" />
+    <Patch name="669 GM GUNSHOT" lbank="0" hbank="0" prog="127" />
+</PatchGroup>
+
+<PatchGroup name="DRUM SET (CATEGORY : 18)">
+    <Patch name="670 STANDARD SET 1" lbank="0" hbank="120" prog="0"  drum="1" />
+    <Patch name="671 STANDARD SET 2" lbank="0" hbank="120" prog="1"  drum="1" />
+    <Patch name="672 STANDARD SET 3" lbank="0" hbank="120" prog="2"  drum="1" />
+    <Patch name="673 STANDARD SET 4" lbank="0" hbank="120" prog="3"  drum="1" />
+    <Patch name="674 STANDARD SET 5" lbank="0" hbank="120" prog="4"  drum="1" />
+    <Patch name="675 DANCE SET 1" lbank="0" hbank="120" prog="26"  drum="1" />
+    <Patch name="676 DANCE SET 2" lbank="0" hbank="120" prog="27"  drum="1" />
+    <Patch name="677 DANCE SET 3" lbank="0" hbank="120" prog="28"  drum="1" />
+    <Patch name="678 DANCE SET 4" lbank="0" hbank="120" prog="29"  drum="1" />
+    <Patch name="679 DANCE SET 5" lbank="0" hbank="120" prog="34"  drum="1" />
+    <Patch name="680 DANCE SET 6" lbank="0" hbank="120" prog="35"  drum="1" />
+    <Patch name="681 TRANCE SET" lbank="0" hbank="120" prog="31"  drum="1" />
+    <Patch name="682 HIP-HOP SET" lbank="0" hbank="120" prog="9"  drum="1" />
+    <Patch name="683 ROOM SET" lbank="0" hbank="120" prog="8"  drum="1" />
+    <Patch name="684 POWER SET" lbank="0" hbank="120" prog="16"  drum="1" />
+    <Patch name="685 ROCK SET" lbank="0" hbank="120" prog="17"  drum="1" />
+    <Patch name="686 ELECTRONIC SET" lbank="0" hbank="120" prog="24"  drum="1" />
+    <Patch name="687 DRUM MACHINE SET 1" lbank="0" hbank="120" prog="25"  drum="1" />
+    <Patch name="688 DRUM MACHINE SET 2" lbank="0" hbank="120" prog="30"  drum="1" />
+    <Patch name="689 DRUM MACHINE SET 3" lbank="0" hbank="120" prog="33"  drum="1" />
+    <Patch name="690 JAZZ SET" lbank="0" hbank="120" prog="32"  drum="1" />
+    <Patch name="691 BRUSH SET" lbank="0" hbank="120" prog="40"  drum="1" />
+    <Patch name="692 ORCHESTRA SET" lbank="0" hbank="120" prog="48"  drum="1" />
+    <Patch name="693 LATIN SET 1" lbank="0" hbank="120" prog="49"  drum="1" />
+    <Patch name="694 LATIN SET 2" lbank="0" hbank="120" prog="50"  drum="1" />
+    <Patch name="695 INDONESIAN SET" lbank="0" hbank="120" prog="53"  drum="1" />
+    <Patch name="696 INDIAN SET" lbank="0" hbank="120" prog="54"  drum="1" />
+    <Patch name="697 ARABIC SET" lbank="0" hbank="120" prog="52"  drum="1" />
+    <Patch name="698 CHINESE SET" lbank="0" hbank="120" prog="51"  drum="1" />
+    <Patch name="699 SFX SET 1" lbank="0" hbank="120" prog="60"  drum="1" />
+    <Patch name="700 SFX SET 2" lbank="0" hbank="120" prog="61"  drum="1" />
+</PatchGroup>
+
+  <Drummaps>
```

Comment:
Hi. The file seems incomplete. 
There is a drummap 'entry' tag but no closing 'entry' tag.
There is a 'drummap' tag but no closing 'drummap' tag.
The file seems to be missing the actual drum entries.
This file would likely fail to be read by our XML reader.

Is there more to come?
Thanks.

Reply:
@terminator356 thank you for the review, there was indeed a xml error, I've removed this part and it is now valid. The drummap seems pretty standard, so I don't think it needs a dedicated section at the moment, it works well this way.